src/actions/index.js

```diff
@@ -0,0 +1,21 @@
+import axios from 'axios';
+
+export const fetchGitRepo = (searchTerm) => {
```

Comment:
Maybe implement an error message when a wrong username is introduced
(Can't differentiate between an empty search bar and an invalid username)

Reply:
Yes, Error message added.